item-search/README.md

```diff
@@ -305,3 +306,15 @@ of this extension can be found in the [sort fragment](../fragments/sort).
 This extension is intended to augment the core ItemCollection responses from the `search` API endpoint with a
 JSON object called `context` that includes the number of items `matched`, `returned` and the `limit` requested.
 The full description and examples of this are found in the [context fragment](../fragments/context).
+
+### Query
```

Comment:
I think there's general consensus that it will be deprecated. I'm not so sure about removing it completely, at least until the CQL support is well-supported on all of our common language/DB combinations.